hive-metastore/src/test/java/org/apache/iceberg/hive/TestCachedClientPool.java

```diff
@@ -28,30 +28,46 @@
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.conf.HiveConf;
 import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.iceberg.CatalogProperties;
 import org.apache.iceberg.CatalogUtil;
 import org.apache.iceberg.exceptions.ValidationException;
 import org.apache.iceberg.hive.CachedClientPool.Key;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-public class TestCachedClientPool extends HiveMetastoreTest {
+public class TestCachedClientPool {
+  private static final long EVICTION_INTERVAL = TimeUnit.SECONDS.toMillis(10);
+  private static final String DB_NAME = "hivedb";
+
+  @RegisterExtension
+  private static final HiveMetastoreExtension HIVE_METASTORE_EXTENSION =
+      new HiveMetastoreExtension(DB_NAME, Collections.emptyMap());
 
   @Test
   public void testClientPoolCleaner() throws InterruptedException {
-    CachedClientPool clientPool = new CachedClientPool(hiveConf, Collections.emptyMap());
+    CachedClientPool clientPool =
+        new CachedClientPool(
+            HIVE_METASTORE_EXTENSION.hiveConf(),
+            ImmutableMap.of(
+                CatalogProperties.CLIENT_POOL_CACHE_EVICTION_INTERVAL_MS,
+                String.valueOf(EVICTION_INTERVAL)));
     HiveClientPool clientPool1 = clientPool.clientPool();
     assertThat(clientPool1)
         .isSameAs(
             CachedClientPool.clientPoolCache()
-                .getIfPresent(CachedClientPool.extractKey(null, hiveConf)));
+                .getIfPresent(
+                    CachedClientPool.extractKey(null, HIVE_METASTORE_EXTENSION.hiveConf())));
     TimeUnit.MILLISECONDS.sleep(EVICTION_INTERVAL - TimeUnit.SECONDS.toMillis(2));
     HiveClientPool clientPool2 = clientPool.clientPool();
     assertThat(clientPool2).isSameAs(clientPool1);
     TimeUnit.MILLISECONDS.sleep(EVICTION_INTERVAL + TimeUnit.SECONDS.toMillis(5));
     assertThat(
             CachedClientPool.clientPoolCache()
-                .getIfPresent(CachedClientPool.extractKey(null, hiveConf)))
+                .getIfPresent(
+                    CachedClientPool.extractKey(null, HIVE_METASTORE_EXTENSION.hiveConf())))
```

Comment:
nit: some test we are extracting to a variable and in some test we are using directly. Let us have a unified style.

Reply:
At other test `hiveConf` was being used at multiple places, that's the reason I created a local variable.